schemas/1.0/general/partner.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$id": "ministryofjustice/laa-criminal-legal-aid-schemas/main/schemas/1.0/general/partner.json",
+  "title": "Partner",
+  "description": "Holds attributes of a partner. Extends from Person.",
+  "type": "object",
+  "allOf": [{ "$ref": "#/definitions/person" }],
+  "properties": {
+    "date_of_birth": { "type": "string", "format": "date" },
+    "nino": { "type": ["string", "null"] },
+    "benefit_type": {"type": ["string", "null"], "enum": ["universal_credit", "guarantee_pension", "jsa", "esa", "income_support", "none", null] },
+    "last_jsa_appointment_date": { "type": ["string", "null"], "format": "date" },
+    "home_address": { "$ref": "#/definitions/address" },
+    "involvement_in_case": {"type": ["string", "null"], "enum": ["victim", "prosecution_witness", "codefendant", "no_involvement", null] },
+    "conflict_of_interest": { "anyOf": [{"type": "null"}, {"type": "string", "enum": ["yes", "no"]}]},
+    "has_same_address_as_client": { "anyOf": [{"type": "null"}, {"type": "string", "enum": ["yes", "no"]}]}
+  },
+  "required":[
+    "date_of_birth", "nino", "home_address", "involvement_in_case", "conflict_of_interest",
+    "has_same_address_as_client"
```

Comment:
Sort of forgetting what `required` does, but in case it matters as a heads up not all of these questions will be asked

Reply:
Ah yes looks like it should be  "date_of_birth", "nino", "involvement_in_case"